### ISSUE TITLE
elegantRL_helloworld 和elegantRL下SAC算法收敛速度差别比较大

### ISSUE THREAD

**Issue by ba277351771** [AI4Finance-Foundation/ElegantRL#115]
elegantRL_helloworld
![image](https://user-images.githubusercontent.com/8056302/156368347-2fe4cc3f-5d95-4538-8acb-3a475dcffb03.png)

elegantRL
![image](https://user-images.githubusercontent.com/8056302/156368606-950c2939-be2c-4775-a11c-97cfadc40ea4.png)

**Comment by hmomin** [comment]
您是否运行了 Elegantrl_helloworld 超过 1.36e+05 步？ 我注意到即使是优雅的版本在达到 1.5e+05 步之前的性能也很差......

**Comment by ba277351771** [comment]
![image](https://user-images.githubusercontent.com/8056302/156476377-525f37cc-ed2b-4cd8-8754-c94e6c3ad8ae.png)

**Comment by ba277351771** [comment]
换了个seed helloworld版还是负分
<img width="693" alt="image" src="https://user-images.githubusercontent.com/8056302/156490479-17a53a43-883a-4ccc-ad99-2ef1bb172991.png">

**Comment by ba277351771** [comment]
![image](https://user-images.githubusercontent.com/8056302/156713310-421877b7-291b-4602-bb54-be35461ed4b7.png)
![image](https://user-images.githubusercontent.com/8056302/156713339-6b1a36c9-8bd9-4fab-9b5a-1fc9f23f000e.png)

**Comment by Yonv1943** [comment]
I had found this bug:

Solution: 
```
class AgentSAC:
    def get_obj_critic_raw(...):
        ...
        next_a, next_log_prob = self.act_target.get_action_logprob(next_s)
        ...
```
should be
```
next_a, next_log_prob = self.act.get_action_logprob(next_s)
```


Why?
Because SAC has not target network for actor. 
So I cancel the `soft_update` in `update_net`. 

```
class AgentSAC:
    def update_net(...):
            # self.soft_update(self.act_target, self.act, self.soft_update_tau) # SAC don't use act_target network

```

**Comment by hmomin** [comment]
Jiahao is right, I can confirm that it fixes the problem (see commit f85de42). It's not perfect, but at least the agent trains properly now...

![bipedal-walker-SAC](https://user-images.githubusercontent.com/72412509/157963264-a697e3f1-5c94-4401-8fde-d47c943274e9.png)
